Not ready for testing. Build is failing: https://jenkins.status.im/job/status-react/job/pull%20requests/job/feature%252Fupgrade-status-go-%25232513/1/console

```
...
  symbol: variable Statusgo
/Users/jenkins/workspace/ure_upgrade-status-go-_2513-UKILAI7KNI3T6ZBAQXBFDYXXFQEFBM7GKY3P6TQMMGYA3HTUBC7Q/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusPackage.java:49: error: cannot find symbol
                return Statusgo.CallRPC(payload);
                       ^
  symbol: variable Statusgo
Note: /Users/jenkins/workspace/ure_upgrade-status-go-_2513-UKILAI7KNI3T6ZBAQXBFDYXXFQEFBM7GKY3P6TQMMGYA3HTUBC7Q/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
16 errors
:react-native-status:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-status:compileReleaseJavaWithJavac'.
```

@adambabik says: Hard to tell it looks like the module is not available at all, but nothing changed in a way we build them

I will debug more later (Monday), but maybe someone has ideas.

--- a/modules/react-native-status/android/build.gradle
+++ b/modules/react-native-status/android/build.gradle
@@ -16,5 +16,5 @@ dependencies {
     compile 'com.facebook.react:react-native:+'
     compile 'com.instabug.library:instabug:3+'
     compile 'status-im:function:0.0.1'
-    compile(group: 'status-im', name: 'status-go', version: 'develop-gad9a877', ext: 'aar')
+    compile(group: 'status-im', name: 'status-go', version: 'develop-g6927638', ext: 'aar')
 }

--- a/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java
+++ b/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java
@@ -11,7 +11,7 @@ import android.webkit.WebStorage;
 
 import com.facebook.react.bridge.*;
 import com.facebook.react.modules.core.DeviceEventManagerModule;
-import com.github.status_im.status_go.cmd.Statusgo;
+import com.github.status_im.status_go.Statusgo;
 
 import java.io.File;
 import java.io.IOException;

--- a/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusPackage.java
+++ b/modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusPackage.java
@@ -5,7 +5,7 @@ import com.facebook.react.bridge.JavaScriptModule;
 import com.facebook.react.bridge.NativeModule;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.uimanager.ViewManager;
-import com.github.status_im.status_go.cmd.Statusgo;
+import com.github.status_im.status_go.Statusgo;
 
 import java.util.ArrayList;
 import java.util.Collections;

--- a/modules/react-native-status/ios/RCTStatus/pom.xml
+++ b/modules/react-native-status/ios/RCTStatus/pom.xml
@@ -25,7 +25,7 @@
                         <artifactItem>
                             <groupId>status-im</groupId>
                             <artifactId>status-go-ios-simulator</artifactId>
-                            <version>develop-gad9a877</version>
+                            <version>develop-g6927638</version>
                             <type>zip</type>
                             <overWrite>true</overWrite>
                             <outputDirectory>./</outputDirectory>

